src/components/manage/Blocks/ToC/View.jsx

```diff
@@ -54,6 +54,7 @@ const View = (props) => {
         const level = entry[0];
         const title = entry[1];
         const items = [];
+        if (!title?.trim()) return;
```

Comment:
Below, we use `title || block.plaintext` as the title of the ToC entry, so I think we should also trim the same thing here, and not only the title?

Reply:
@davisagli Indeed, thank you for noticing this, I've fixed it.